theme/static/style/style.css

```diff
@@ -463,8 +495,22 @@ pre code {
         padding-left: 1.8rem;
         box-sizing: border-box;
     }
-}
 
+    /*features page*/
+    .features .w50 {
+        min-width: 95%;
+        max-width: 95%;
```

Comment:
I think something like width: 100% and padding (or margin) 1.8rem on both sides, probably combined with box-sizing: border-box will have the same resizing effect and allow all the elements to have the same alignment.